src/Altinn.App.Core/Features/Notifications/Email/EmailNotificationClient.cs

```diff
@@ -76,7 +70,8 @@ public async Task<EmailOrderResponse> Order(EmailNotification emailNotification,
                 if (orderResponse is null)
                     throw new JsonException("Couldn't deserialize email notification order response.");
 
-                Telemetry.OrderCount.WithLabels(Telemetry.Types.Email, Telemetry.Result.Success).Inc();
+                _telemetry?.RecordNotificationOrder(_orderType, Telemetry.Notifications.OrderResult.Success);
+                activity?.SetStatus(ActivityStatusCode.Ok);
```

Comment:
Avoid using status in places like this for now. Status error will be set on the root trace, it's unclear if we need it here (and if we did, we should do it a lot more places as well)